src/core/css/inc/vars.scss

```diff
@@ -15,6 +15,8 @@ $button-background-color: #0095dd;
 $button-background-disabled-color: #8a9ba8;
 $button-background-hover-color: #08c;
 
+$list-item-background-color: #f5f5f5;
```

Comment:
I'm tempted to say this should have a name like `$background-accent` since I could see it being used for non-list-items.